ansible_collections/arista/avd/roles/eos_designs/doc/l3ls-evpn/network-services.md

```diff
@@ -207,8 +210,6 @@ tenants:
               always: < true | false >
             update_source: < interface >
             ebgp_multihop: < 1-255 >
-            address_family:
-              - < ipv4 | ipv6 >
             # Nodes is required to restrict configuration of BGP neighbors to certain nodes in the network.
             nodes: [ < node_1 >, < node_2> ]
             # Next hop settings can be either ipv4 or ipv6 for one neighbor, this will be applied by a uniquely generated route-map per neighbor.
```

Comment:
Add route-map-in, route-map-out and password

Reply:
this has been added to the documentation.